src/resssources/HeaderParagraph.rs

```diff
@@ -0,0 +1,10 @@
+pub struct HeaderParagraph {
+    format: &str,
```

Comment:
it could be a tuple of the http and https variant of the link
And there is some other rare variants from old days: https://github.com/air-balloon/debian-dashboard/blob/09cfd8eb2a435c3965b08ebdcc7061ee87ba4540/src/UDD.php#L449

Reply:
Impossible to do for now (as I know) with a tuple/enum we must do it outside the struct